src/components/pages/ClubsPages/anderson.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { LayoutContainer } from '../../common';
+import NavBar from '../../common/NavBar';
+import NavMenu from '../../common/NavMenu';
+import { Layout, Card, Dropdown, Button, Menu } from 'antd';
+import { DownOutlined, StockOutlined } from '@ant-design/icons';
+import Plot from 'react-plotly.js';
+import { getClubs, getFeedback } from '../../../state/actions';
+import { AdminContext } from '../../../state/contexts';
+import axios from 'axios';
+import '../ClubPages/anderson.css';
+
+const { Content, Sider } = Layout;
+
+function Anderson() {
+  const context = useContext(AdminContext);
+  const [whichClub, setWhichClub] = useState('Anderson');
+  const [authtoken, setAuthtoken] = useState('');
+  const [plotData, setPlotData] = useState('');
+
+  useEffect(() => {
+    graph();
+    getFeedback('authState', context);
+    if (context.clubs.length === 0) {
+      getClubs('authState', context);
+    }
+  }, []);
+
+  //   const menu = (
```

Comment:
Because I was working with Robert on this, I know this is temporary code that will be removed when the new page is completed.